web/theme/default/css/xibo.css

```diff
@@ -1613,4 +1613,266 @@ div.dataTables_wrapper div.dataTables_info {
+    padding: 9px 20px;
+    font-size: 20px;
+    font-weight: bold;
+    transition: background-color 0.3s ease;
+    text-decoration: none !important;
+}
+
+
+.btn-orange {
+    background-color: #EB7857;
+    color: #fff;
+    border: none;
+}
+
+.btn-orange:hover {
+    background-color: #BC6046;
+    color: #fff;
+}
+
+.btn-outlined {
+    background-color: #fff;
+    color: #0E70F6;
+    border: 2px solid #0E70F6;
+}
+
+.btn-outlined:hover {
+    background-color: #0E70F6;
+    color: #fff;
+}
+
+.onboarding-cards-container {
+    display: flex;
+    flex-wrap: wrap;
+    gap: 20px;
+    margin-top: 80px;
+    position: relative;
+}
+
+.onboarding-card {
+    display: flex;
+    flex-direction: column;
+    border-radius: 8px;
+    height: 160px;
+    width: 234px;
+    padding: 20px;
+    box-shadow: 0px 5px 30px 0px rgba(0, 0, 0, 0.10);
+    justify-content: space-between;
+    color: #0E70F6;
+    transition: background-color 0.3s ease;
+    background-color: #FFFFFF;
+    z-index: 2;
+}
+
+
+.onboarding-card  h3{
+    font-size: 24px;
+    line-height: 16px;
+    margin-bottom: 0;
+    font-weight: 600;
+}
+
+.onboarding-card:hover{
+    background-color: #0E70F6;
+    color: #ffffff;
+}
+
+.onboarding-welcome-image{
+    border-radius: 20px;
+    box-shadow: 0px 5px 30px 0px rgba(0, 0, 0, 0.10);
+    width: 100%;
+    height: 500px;
+    object-fit: cover;
+}
+
+.service-card {
+    display: flex;
+    padding: 40px;
+    column-gap: 86px;
+    border-radius: 20px;
+    box-shadow: 0px 5px 30px 0px rgba(0, 0, 0, 0.10);
+    background-color: #FFFFFF;
+    z-index: 2;
+}
+
+.service-card h3 {
+    color: #0E70F6;
+    font-size: 32px;
+}
+
+.service-card p {
+    color: #484848;
+    font-size: 24px;
+    margin-bottom: 0;
+}
+
+.service-card .text-link {
+    text-decoration: underline !important;
+    font-weight: 600 !important;
+    color: #484848;
+}
+
+
+
+.service-card-image {
+    width: 274px;
+    height: 154px;
+}
+
+.service-card-text {
+    display: flex;
+    flex-direction: column;
+    row-gap: 20px;
+}
+
+.line-curve {
+    position: absolute;
+    z-index: 0;
+    right: -50%;
+    bottom: -50%;
+    height: 137.9px !important;
+}
+
+.others-card-container{
+    margin-top: 40px;
+    display: flex;
+    flex-wrap: wrap;
+    justify-content: space-between;
+}
+
+.others-card {
+    display: flex;
+    flex-direction: column;
+    padding: 24px;
+    row-gap: 15px;
+    background-color: #fff;
+    border-radius: 20px;
+    box-shadow: 0px 5px 30px 0px rgba(0, 0, 0, 0.10);
+    align-items: start;
+    width: 480px;
+    height: 312px;
+}
+
+.others-card h3 {
+    font-size: 24px;
+    font-weight: 600;
+    color: #1775F6;
+}
+
+.others-card p {
+    font-size: 16px;
+    color: #484848;
+    line-height: normal
+}
+
+.others-card a{
+    color: #1775F6;
+    font-size: 16px;
+    font-weight: 500;
+}
+
+.links-list{
+    display: flex;
+    column-gap: 24px;
+    margin-top: auto;
+}
+
+.optional-link{
```

Comment:
To make the CSS consistent, can we have a space between the rule and the curly bracket?

`.optional-link {` instead of `.optional-link{`